ValidationLibrary.AzureFunctions/Startup.cs

```diff
@@ -76,9 +85,14 @@ public void Configure(IWebJobsBuilder builder)
                 .AddTransient<IValidationClient, ValidationClient>()
                 .AddSingleton(provider =>
                 {
-                    var rules = validationRules.Select(r => (IValidationRule)provider.GetService(r)).ToArray();
+                    var logger = provider.GetService<ILogger<ValidationLibrary.RepositoryValidator>>();
```

Comment:
This is not `ValidationLibrary.RepositoryValidator` logigng, this is `Startup`-logging

Reply:
Ah, like that?

Reply:
yes